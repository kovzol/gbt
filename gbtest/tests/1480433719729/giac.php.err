// Using locale /usr/share/locale/
// C
// /usr/share/locale/
// giac
// UTF-8
// Maximum number of parallel threads 8
Help file /usr/share/giac/doc/local/aide_cas not found
Added 0 synonyms

// Groebner basis computation time 0.002297 Memory 0.024468M
// Time 0
0.04user 0.00system 0:00.04elapsed 100%CPU (0avgtext+0avgdata 24504maxresident)k
0inputs+16outputs (0major+1573minor)pagefaults 0swaps
