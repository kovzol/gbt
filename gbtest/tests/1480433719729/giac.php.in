gbasis([77-25*v2,v8^2-1*v6^2-1*v5^2+2*v6*v2-1*v2^2+2*v5*v1-1*v1^2,-93-100*v4,3+2*v1,v7^2-1*v6^2-1*v5^2+2*v6*v4-1*v4^2+2*v5*v3-1*v3^2,-223-100*v3,625-50*v8^2+v8^4-50*v7^2-2*v8^2*v7^2+v7^4],[v8,v7,v4,v3,v2,v1,v5,v6],revlex)