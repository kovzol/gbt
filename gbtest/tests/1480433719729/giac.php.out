Welcome to giac readline interface
(c) 2001,2016 B. Parisse & others
Homepage http://www-fourier.ujf-grenoble.fr/~parisse/giac.html
Released under the GPL license 3.0 or above
See http://www.gnu.org for license details
May contain BSD licensed software parts (lapack, atlas, tinymt)
-------------------------------------------------
Press CTRL and D simultaneously to finish session
Type ?commandname for help
0>> gbasis([77-25*v2,v8^2-1*v6^2-1*v5^2+2*v6*v2-1*v2^2+2*v5*v1-1*v1^2,-93-100<*v6^2-1*v5^2+2*v6*v2-1*v2^2+2*v5*v1-1*v1^2,-93-100*                         v4,3+2*v1,v7^2-1*v6^2-1*v<2+2*v5*v1-1*v1^2,-93-100*v4,3+2*v1,v7^2-1*v6^2-1*v5                         ^2+2*v6*v4-1*v4^2+2*v5*v3<4,3+2*v1,v7^2-1*v6^2-1*v5^2+2*v6*v4-1*v4^2+2*v5*v3-                         1*v3^2,-223-100*v3,625-50<2+2*v6*v4-1*v4^2+2*v5*v3-1*v3^2,-223-100*v3,625-50*                         v8^2+v8^4-50*v7^2-2*v8^2*<*v3^2,-223-100*v3,625-50*v8^2+v8^4-50*v7^2-2*v8^2*v                         7^2+v7^4],[v8,v7,v4,v3,v2<8^2+v8^4-50*v7^2-2*v8^2*v7^2+v7^4],[v8,v7,v4,v3,v2,                         v1,v5,v6],revlex)
[2446710000*v8^2-585460000*v5*v6-1554720000*v6^2+2415467800*v5+12062072200*v6-23242654293,2446710000*v7^2-585460000*v5*v6-1554720000*v6^2-1156728800*v5-7560542000*v6-8810490687,2446710000*v5^2-585460000*v5*v6+891990000*v6^2+9755597800*v5-3009661400*v6+5472912951,100*v4+93,100*v3+223,25*v2-77,2*v1+3]
1>> 